holiday-calendar-base/src/main/lombok/com/github/davejoyce/calendar/SpecialAnniversary.java

```diff
@@ -34,6 +37,8 @@
  */
 public class SpecialAnniversary extends Holiday {
 
+    @Getter
+    @NonNull
```

Comment:
Similar comment here for `@NonNull`

Reply:
Added `@NonNull` to the constructor parameter